CONTRIBUTING.txt

```diff
@@ -335,7 +335,7 @@ detailing the test coverage::
 Activate Travis-CI for your fork (optional)
 -------------------------------------------
 
-Travis-CI checks all unittests in the project to prevent breakage.
+Travis-CI checks all unit tests in the project to prevent breakage.
```

Comment:
I'd keep the older version here (reference to Python `unittest` module), otherwise :+1:

Reply:
Sounds good 👍

Reply:
I think the suggested change is correct. A unit test is a thing, the module is one word because they didn't want dashes or underscores.